.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: elixir
+
+elixir: '1.5'
```

Comment:
Not sure if we want a newer OTP? Travis seems to give us version 18.

Reply:
The version of OTP we use really shouldn't matter. The Erlang version does (there are a few versions that don't do HTTPS properly), but whatever we get on Travis seems to be okay.